spec/temporal.html

```diff
@@ -4,4 +4,126 @@
 <emu-clause id="sec-temporal-object">
   <h1>The Temporal Object</h1>
   <p><mark>TODO: Specify.</mark></p>
+  <p>
+    The Temporal object is the %Temporal% intrinsic object and the initial value of the Temporal property of the global object.
+    The Temporal object is a single ordinary object.
+  </p>
+  <p>
+    The value of the [[Prototype]] internal slot of the Temporal object is the intrinsic object %ObjectPrototype%.
+  </p>
+  <p>
+    The Temporal object is not a function object.
+    It does not have a [[Construct]] internal method; it is not possible to use the Temporal object as a constructor with the new operator.
+    The Temporal object does not have a [[Call]] internal method; it is not possible to invoke the Temporal object as a function.
+  </p>
+
+  <emu-clause id="sec-temporal-properties">
+    <h1>Properties of the Temporal Object</h1>
+    <emu-clause id="sec-temporal.now">
+      <h1>Temporal.now</h1>
+      <p>
+        The initial value of `Temporal.now` is %Temporal.now%.
+      </p>
+      <p>
+        This property has the attributes { [[Writable]]: false, [[Enumerable]]: false, [[Configurable]]: false }.
+      </p>
+    </emu-clause>
+  </emu-clause>
+</emu-clause>
+
+<emu-clause id="sec-temporal-now">
+  <h1>The Temporal.now Object</h1>
+  <p>
+    The Temporal.now object is the %Temporal.now% intrinsic object.
```

Comment:
Should this also say that it is an ordinary object and its prototype is Object.prototype, as above with the Temporal object?